README.md

```diff
@@ -21,7 +21,7 @@ scripts/infra/cloud-instance.sh ec2 ssh
 # Regardless of how you setup your instance
 git clone https://github.com/instructlab/taxonomy.git && pushd taxonomy && git branch rc && popd
 git clone --bare https://github.com/instructlab/eval.git && git clone eval.git/ && cd eval && git remote add syncrepo ../eval.git
-python -m venv venv
+python3 -m venv venv
```

Comment:
`python` outside a virtualenv doesn't always exist, or point to `python3`. Better to be explicit.

Reply:
seems out of scope of this PR, although it's not a big one so I'm not opposed to including it if you don't think it's an issue

Reply:
Sorry, yes. It sneaked in. I can drop it or split out into a separate commit.